README.md

```diff
@@ -245,6 +247,14 @@ the [Lombok Gradle Plugin](https://plugins.gradle.org/plugin/io.freefair.lombok)
 to delombok your source code before it is passed to the Checker Framework
 for typechecking. This plugin does not support any other use of Lombok.
 
+## Compatibility with other annotation processors
+
+The plugin should be compatible with other annotation processors, as long
+as they are run via `javac`'s auto-discovery mechanism. If you already provide
+a `-processor` option to `javac`, you **cannot** use this plugin: you
+must also provide the checkers you wish to use by hand, because the `-processor`
+flag to `javac` disables annotation processor auto-discovery.
```

Comment:
If you keep a note about the implementation via auto-discovery, this would be the place to put it.  Right now this sentence is confusing because it's not clear why it brings up auto-discovery (that's not clear even given the first sentence, which should be deleted in any event).